cmake/install.cmake

```diff
@@ -1,47 +1,47 @@
 ### kagome_install should be called right after add_library(target)
```

Comment:
I think that these changes are added to the index inattentively, right?

Reply:
yes, exactly